go.mod

```diff
@@ -4,89 +4,50 @@ go 1.13
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e // indirect
-	github.com/Masterminds/semver/v3 v3.0.1
+	github.com/Masterminds/semver/v3 v3.0.2
 	github.com/Masterminds/sprig/v3 v3.0.0
-	github.com/Masterminds/vcs v1.13.0
-	github.com/Microsoft/go-winio v0.4.12 // indirect
+	github.com/Masterminds/vcs v1.13.1
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
-	github.com/containerd/containerd v1.3.0
+	github.com/containerd/containerd v1.3.1
 	github.com/cyphar/filepath-securejoin v0.2.2
-	github.com/deislabs/oras v0.7.0
+	github.com/deislabs/oras v0.8.0
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/docker/docker v1.4.2-0.20181221150755-2cb26cfe9cbf
+	github.com/docker/docker v0.7.3-0.20190826074503-38ab9da00309
```

Comment:
I'm curious why there appears to be a version downgrade for docker/docker. Do you know why this is the case?

Reply:
Its very confusing to me.. you will notice that there used to be a replace to the `v0.7.3` version which is now removed:
```
github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
```

Reply:
based on the date this version is newer, no?(20181221150755 -> 20190826074503)